src/styles/index.scss

```diff
@@ -14,14 +15,38 @@ body {
 }
 
 header {
+  padding: 3em 2em;
+
+  a {
+    &,
+    &:visited,
+    &:active {
+      color: #FFF;
+      text-decoration: none;
+    }
+
+  }
+
   div {
+    color: #FFF;
     display: inline;
+    padding-right: 2em;
   }
+}
 
-  a {
-    color: #000;
-    text-decoration: none;
-  }
+.header-text {
+  float: left;
+}
+
+.header-links {
+  float: right;
+}
+
+.button-login {
+  background-color: #F00;
+  border-radius: 1em;
+  color: #FFF;
+  padding: 1em;
 }
```

Comment:
`.header-text` , `.header-links` , and `.button-login` can be be nested in your header selector